sktime/forecasting/base/adapters/_pytorchforecasting.py

```diff
@@ -314,6 +327,11 @@ def _predict(
                 else None
             ),
         )
+        try:
```

Comment:
Can you extract this check into a private method and use it also in line 310-318? 

Is there a reason why you haven't used `torch.manual_seed` here?

Reply:
I am not sure how to get the original seed, and state is more precise than a seed I think.